### PR TITLE
Xian chap12

### PR DESCRIPTION
**Pull request by zhxianlucky**

### CODE REVIEW

File: Textbook/第12章-人工智能安全与隐私/12.2-人工智能保密性.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+目前有多种达成差分隐私机制，例如适用于数值型输出的Laplace机制，其对返回结果添加Laplace分布的噪声；还有适用于非数值输出的指数机制，其以一定的随机概率从输出范围中选择输出；还有所谓的组合机制，可以将多个随机算法组合后达到相应的隐私效果。下面我们以随机梯度下降（Stochastic Gradient Descent)算法为例，论述如何保证模型参数不包含个体隐私信息，该算法由Abadi等人在2016年提出：
+
+- 第一步，计算出梯度值，并根据梯度大小决定噪声的大小。
+- 第二步，通过对梯度进行削减，使噪声规模不影响精度。
+- 第三步，利用差分隐私组合规则计算隐私成本ε
+- 第四步，通过调整超参数平衡隐私、准确率和性能
+
+<center> <img src="./img/2/12-2-1-dif.png" width="280" height="180" /></center>
+<center>图12-2-1. 带差分隐私的SGD算法</center>
```
Comment (YanjieGao) [inline Textbook/第12章-人工智能安全与隐私/12.2-人工智能保密性.md:43]: Hi Xian, does this figure need citation?

File: Textbook/第12章-人工智能安全与隐私/12.2-人工智能保密性.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+<!--Copyright © Microsoft Corporation. All rights reserved.
+  适用于[License](https://github.com/microsoft/AI-System/blob/main/LICENSE)版权许可-->
+
+# 12.2 人工智能保密性
+
```
Comment (YanjieGao) [inline Textbook/第12章-人工智能安全与隐私/12.2-人工智能保密性.md:5]: In the next iteration, how about adding a bit tiny examples in 12.2 and 12.3 like 图12-1-1. 对抗样本的直观表示, etc. Very useful for "初学者" to understand the context and concept.

File: Textbook/README.md [hunk 3]
```diff
@@ -246,14 +246,14 @@
   - 11.3 应用：
     - 11.3.1 预训练模型稀疏化
     - 11.3.2 移动端模型稀疏化
-- [12. 安全与隐私]()
+- [12. 人工智能安全与隐私]()
```
Comment (YanjieGao) [inline Textbook/README.md:249]: let's add link to the content in outline

File: Textbook/第12章-人工智能安全与隐私/12.2-人工智能保密性.md (hunk 1 above)
Comment (zhxianlucky) [reply Textbook/第12章-人工智能安全与隐私/12.2-人工智能保密性.md:43]: I adapt the latex code of the original paper to a Chinese version. I will add a citation later.

File: Textbook/第12章-人工智能安全与隐私/12.2-人工智能保密性.md (hunk 2 above)
Comment (zhxianlucky) [reply Textbook/第12章-人工智能安全与隐私/12.2-人工智能保密性.md:5]: Sure, I will check and add some figures.

File: Textbook/README.md (hunk 3 above)
Comment (zhxianlucky) [reply Textbook/README.md]: Done.

**Review by YanjieGao — APPROVED** [review]

**Pull request merged**